lib/uklock/include/uk/rwlock.h

```diff
@@ -27,7 +27,6 @@
 #define UK_RW_READERS(x)			((x) >> READERS_SHIFT)
 
 #define UK_RW_UNLOCK				(UK_RW_READERS_LOCK(0))
-#define OWNER(x)					((x) & ~FLAG_MASK)
```

Comment:
A couple of notes for the commit message:

* "the owner and flags for the reader writer lock were store" => "the owner and flags for the reader writer lock were stored"
* Could add in the commit description a reference to the commit that updated the stack alignment e.g. commit title and id.
* "separates the owner and flag field" => "separates the owner and flag fields"
* "new implementation uses pointer to the `struct uk_thread` as the owner of the lock" => "the new implementation uses a pointer to the `struct uk_thread` as the owner of the lock."